corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -657,7 +673,7 @@ hqDefine('geospatial/js/models', [
             createActivePolygonLayer(polygonObj);
 
             self.activeSavedPolygon = polygonObj;
-            self.addPolygonsToFilterList(polygonObj.geoJson.features);
```

Comment:
is this method not needed anymore?

Reply:
This method still gets used by the "Case Grouping" page [here](https://github.com/dimagi/commcare-hq/blob/e673e475f7eb20c1495a02a3ca56d805022f7616/corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js#L543) and [here](https://github.com/dimagi/commcare-hq/blob/e673e475f7eb20c1495a02a3ca56d805022f7616/corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js#L549). It is being used to add polygons to the URL when a user draws/moves/deletes a new polygon on the map.